app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/fragments/mvp/FragmentDialogView.kt

```diff
@@ -8,10 +8,10 @@ import android.widget.Toast
 import com.puzzlebench.clean_marvel_kotlin.R
 import com.puzzlebench.clean_marvel_kotlin.presentation.DOT
 import com.puzzlebench.clean_marvel_kotlin.presentation.MainActivity
-import com.puzzlebench.clean_marvel_kotlin.presentation.NO_DESCRIPTION
 import com.puzzlebench.clean_marvel_kotlin.presentation.SCREEN_PERCENTAGE
 import com.puzzlebench.clean_marvel_kotlin.presentation.extension.getImageByUrl
 import com.puzzlebench.clean_marvel_kotlin.presentation.fragments.CharacterFragmentDialog
+import com.puzzlebench.clean_marvel_kotlin.presentation.fragments.mvp.contracts.FragmentDialogContracts
```

Comment:
dame

Reply:
the view implements the FragmentDialogContracts